lib/kafo/logger.rb

```diff
@@ -105,7 +105,7 @@ def dump_errors
 
     def self.dump_buffer(buffer)
       buffer.each do |log|
-        self.loggers.each { |logger| logger.send log[0], *log[1], &log[2] }
+        self.loggers.each { |logger| logger.send log[0], *([log[1]].flatten(1)), &log[2] }
```

Comment:
This was breaking tests on Ruby 1.8.7. In some of the log buffer records the messages were strings and not arrays and the old logger didn't recover from that.